pkg/config/process.go

```diff
@@ -69,4 +93,25 @@ func setupProcesses(config Config) {
 		"DD_PROCESS_AGENT_DISCOVERY_ENABLED",
 	)
 	procBindEnvAndSetDefault(config, "process_config.process_discovery.interval", 4*time.Hour)
+
+	AddOverrideFunc(loadProcessTransforms)
+}
+
+// loadProcessTransforms loads transforms associated with process config settings. This is used to handle deprecated settings
+func loadProcessTransforms(config Config) {
+	if config.IsSet("process_config.enabled") {
+		log.Debug("process_config.enabled is deprecated, use process_config.container_collection.enabled" +
```

Comment:
I think this should be `Warning` or `Info`. Also do we have a documentation somewhere that we can link here explaining this change ?

Reply:
I agree, it probably should be higher than debug. I will change it to Info. That way we can still change it to `Warn` when the helm chart and operator catch up

As far as documentation is concerned, we have the yaml template that we could link to, and we plan to update https://docs.datadoghq.com/infrastructure/process/, once 7.35 is released